app/mapping.js

```diff
@@ -38,7 +39,8 @@ var codeMapping = {
   "pp": "partial page",
   "ed": "engine disabled",
   "b": "blocked, IP address is blocked",
-  "uc": "User Canceled, request was cancelled by the user"
+  "uc": "User Canceled, request was cancelled by the user",
+  "pl": "Preload headers"
```

Comment:
one code styling thingie: we should use trailing commas so we would have one diff here.
consequently, I'd suggest adding prettier to this repo and using the this .prettierrc file as a start:
https://github.com/fasterize/insight/blob/master/.prettierrc